pkg/restapi/vc/operation/operations.go

```diff
@@ -243,14 +243,12 @@ func (o *Operation) storeVCHandler(rw http.ResponseWriter, req *http.Request) {
 	doc.Content["message"] = data
 	doc.ID = data.Credential.ID
 
-	locationOfDocument, err := o.client.CreateDocument(data.Profile, &doc)
+	_, err = o.client.CreateDocument(data.Profile, &doc)
 	if err != nil {
 		o.writeErrorResponse(rw, http.StatusBadRequest, err.Error())
 
 		return
 	}
-
-	o.writeResponse(rw, locationOfDocument)
```

Comment:
As a result of my above comment, no response is needed other than http.StatusOK (which is written automatically)